packages/block-library/src/video/edit.js

```diff
@@ -97,9 +97,15 @@ function VideoEdit( {
 			setAttributes( { src: undefined, id: undefined } );
```

Comment:
Do I need to clear `poster` here as well?

Reply:
Decided that since it's getting set here, makes sense to clear it. Less likely to end up with an error condition where there's still a poster set, but the other information is missing.